pyo365/drive.py

```diff
@@ -811,14 +868,16 @@ def create_child_folder(self, name, description=None):
         if not self.object_id:
             return None
 
-        url = self.build_url(self._endpoints.get('list_items').format(id=self.object_id))
+        url = self.build_url(self._endpoints.get(
+            'list_items').format(id=self.object_id))
 
-        data = {'name': name}
+        data = {'name': name,
+                'folder': {}}
         if description:
             data['description'] = description
 
         response = self.con.post(url, data=data)
-        if not response:
+        if not response.ok:
```

Comment:
`response.ok` is the same as `bool(response)`.

so it's more pythonic to use `if not response:`

Reply:
Okay ;)